azure-devops/cross-build-test.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+parameters:
+- name: hostArch
+  type: string
+  default: amd64
+- name: targetPlatform
+  type: string
+- name: vsDevCmdArch
+  type: string
+- name: buildOutputLocationVar
+  type: string
+  default: buildOutputLocation
+
+  - template: azure-devops/cross-build-test.yml

```

Comment:
Why are you recursively including this file?

You probably meant to include `cross-build.yml` and then `cross-test.yml`.